content/en/docs/concepts/services-networking/service.md

```diff
@@ -684,15 +684,16 @@ The set of protocols that can be used for LoadBalancer type of Services is still
 
 #### Disabling load balancer NodePort allocation {#load-balancer-nodeport-allocation}
 
-{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
 
 Starting in v1.20, you can optionally disable node port allocation for a Service Type=LoadBalancer by setting
 the field `spec.allocateLoadBalancerNodePorts` to `false`. This should only be used for load balancer implementations
 that route traffic directly to pods as opposed to using node ports. By default, `spec.allocateLoadBalancerNodePorts`
 is `true` and type LoadBalancer Services will continue to allocate node ports. If `spec.allocateLoadBalancerNodePorts`
 is set to `false` on an existing Service with allocated node ports, those node ports will NOT be de-allocated automatically.
```

Comment:
When features graduate to beta, and I'm reviewing, I also look for and fix any style nits. For alpha features we're a bit less strict.
```suggestion
is set to `false` on an existing Service with allocated node ports, those node ports will **not** be de-allocated automatically.
```